src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * The parent class for Email, Address and phone.
```

Comment:
Couple of things:
- the comment need to tell about the child classes. 
- the comment should be in active form
  - simple `Represents a person's contact` would suffice